crates/zed/src/main.rs

```diff
@@ -1518,11 +1518,20 @@ fn dump_all_gpui_actions() {
     .unwrap();
 }
 
-#[cfg(windows)]
+#[cfg(target_os = "windows")]
 fn check_for_conpty_dll() {
-    use windows_sys::{Win32::System::LibraryLoader::LoadLibraryW, w};
-    let hmodule = unsafe { LoadLibraryW(w!("conpty.dll")) };
-    if hmodule.is_null() {
+    use windows::{
+        Win32::{Foundation::FreeLibrary, System::LibraryLoader::LoadLibraryW},
+        core::w,
+    };
+
+    if let Ok(hmodule) = unsafe { LoadLibraryW(w!("conpty.dll")) } {
+        unsafe {
+            FreeLibrary(hmodule)
```

Comment:
I wonder is the lack of similar call in alacritty going to be a problem for us I wonder?